plugins/virtualization/kvm_cpu

```diff
@@ -57,9 +57,12 @@ def clean_vm_name(vm_name):
 def detect_kvm():
     ''' Check if kvm is installed
     '''
-    kvm = Popen("which kvm", shell=True, stdout=PIPE)
-    kvm.communicate()
-    return not bool(kvm.returncode)
+    if os.path.isfile("/usr/libexec/qemu-kvm"):
+        return True
+    else:
+        kvm = Popen("which kvm", shell=True, stdout=PIPE)
+        kvm.communicate()
+        return not bool(kvm.returncode)
```

Comment:
I didn't change the original behavior. Just added one that fits more for RHEL/CentOS.
In general this part is from my point of view problematic in 2 cases:

1. which lookups are expensive
2. we're not only looking at kvm.

I can change it in general, but as I said the intend of my PR was not to change the original behavior. But I can come up with general improvements, if people would like to see that.